.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.11",
```

Comment:
assume all the requirements are available on 3.11?

Reply:
Yes

Reply:
fwiw: I think having a duplicate docker image as the devcontainer isn't worth the effort. `make serve-local` has hot reload so I don't see any reason to use a devcontainer